nemoguardrails/library/injection_detection/yara_rules/code.yara

```diff
@@ -0,0 +1,61 @@
+rule import_shells
+{
+    meta:
+        author = "Erick Galinkin"
+        description = "Detect import of the os, cmd, subprocess, shutil modules"
+        date = "2025-03-27"
```

Comment:
Did you write these from scratch or adapt them from somewhere? I'm not familiar with yara. If they're new, how should they be evaluated?